myia/utils/misc.py

```diff
@@ -435,3 +435,31 @@ def assert_scalar(*args):
                 raise TypeError(msg)
         elif not isinstance(x, (int, float, np.number)):
             raise TypeError(f'Expected scalar, not {type(x)}')
+
+
+__consolidate__ = True
+__all__ = [
+    'ClosureNamespace',
+    'ErrorPool',
+    'Event',
+    'Events',
+    'HasDefaults',
+    'MISSING',
+    'ModuleNamespace',
+    'NS',
+    'Named',
+    'Namespace',
+    'Registry',
+    'T1',
+    'T2',
+    'TaggedValue',
+    'UNKNOWN',
+    'assert_scalar',
+    'core',
+    'dataclass_fields',
+    'is_dataclass_type',
+    'keyword_decorator',
+    'list_str',
+    'repr_',
+    'resolve_from_path',
```

Comment:
A bunch of those should not be here.

Reply:
T1 and T2 don't belong, but the rest seems fine.